### PR TITLE
add R.mean and R.median

### PR DESCRIPTION
**Pull request by davidchambers**
I'd also like to add a [mode][1] function, but since it should work with values other than numbers it's more of a cousin than a sibling of `R.mean` and `R.median`. I'll open a separate pull request for that function if and when this pull request is merged.

[1]: http://en.wikipedia.org/wiki/Mode_%28statistics%29

### CODE REVIEW

File: src/mean.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * @category Math
+ * @sig [Number] -> Number
+ * @param {Array} xs
+ * @return {Number}
+ * @example
+ *
+ *      R.mean([2, 7, 9]); //=> 6
+ *      R.mean([]); //=> NaN
+ */
+module.exports = _curry1(function mean(xs) {
```
Comment (CrossEye) [inline src/mean.js]: Usually we'd use the name `list`.  Did you want `xs` to distinguish a list of numeric values

**Comment by CrossEye** [comment]
These look good.  `mode` is to my mind significantly more challenging.  How do you capture equality for non-primitives?  What do you do when there is more than one cohort of maximal value?

But that would be for another PR... :herb:

**Comment by buzzdecafe** [comment]
:cow2: :accept:

File: src/mean.js (hunk 1 above)
Comment (davidchambers) [inline src/mean.js]: :zap:

**Comment by davidchambers** [comment]
> But that would be for another PR...

#981

**Pull request merged**